src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1530,6 +1530,34 @@ def receive_timeseries_from_csv(
         compute_timeseries_properties(dict_asset)
 
 
+def replace_nans_in_timeseries_with_0(dict_asset):
+    """
+    Replaces nans in the timeseries (if any) with 0
```

Comment:
Add a blank line between the description and the parameters.

Reply:
@mahendrark In some of the functions, I didn't see the blank line between description and parameters. I haven't seen uniformity.